components/automate-cli/cmd/chef-automate/migration_pg.go

```diff
@@ -44,7 +44,6 @@ var NEW_BIN_DIR = "/hab/pkgs/core/postgresql13/13.5/20220311204618/bin"
 const (
 	AUTOMATE_VERSION            = "3"
 	NEXT_AUTOMATE_VERSION       = "4"
-	AUTOMATE_PG_MIGRATE_LOG_DIR = "/tmp"
 	OLD_PG_VERSION              = "9.6"
 	OLD_PG_DATA_DIR             = "/hab/svc/automate-postgresql/data/pgdata"
 	NEW_PG_DATA_DIR             = "/hab/svc/automate-postgresql/data/pgdata13"
```

Comment:
we can remove the migration_pg.go  changes 
this will be for upgrade from 2.x to 3.x